middleware/getTime.js

```diff
@@ -0,0 +1,7 @@
+const getTime = (date) => {
```

Comment:
You might consider using a library like https://date-fns.org/ instead of this. It's too much custom code for simple date formatting, in my opinion.

Reply:
I use `date-and-time` library because it's light-weighted and I like their documentation better. I also move the getTime logic to my model by setting it as computed properties